Automattic/woocommerce-services
# Change jquery.load to jquery.on('load')

Closes https://github.com/Automattic/woocommerce-services/issues/2155

### Description
Since jquery 1.8, `jquery.load` is deprecated. https://api.jquery.com/load-event/ Our jquery version is `1.12.4` so we need to fix that.

### How to test
1. In `wordpress` database, look for `meta_key like '%dismissed_wp_pointers%'` in the `wp_usermeta` table
2. Remove your entry
3. You should see the pointer to show again
![image](https://user-images.githubusercontent.com/572862/92182114-1eac9f80-ee08-11ea-84e2-b9c1f6ca6198.png)
